### ISSUE TITLE
EventsManager.EnterpriseEventsAsync fails with RFC3339DateFormat

### ISSUE THREAD

**Issue by kendomen** [box/box-windows-sdk-v2#310]
I was trying this:

```
DateTime createdAfter = DateTime.Parse("2016-04-11T00:00:00");
Task<BoxEventCollection<BoxEnterpriseEvent>> t = adminClient.EventsManager.EnterpriseEventsAsync(100, null, null, createdAfter, null );
```
         
And go the error:  "Invalid create after format".

It looks like the error can be solved by updating BoxEventsManager.EnterpriseEventsAsync like this...

```
// bug
//var createdAfterString = createdAfter.HasValue ? createdAfter.Value.ToString(Constants.RFC3339DateFormat) : null;  
            
// fix
var createdAfterString = createdAfter.HasValue ? createdAfter.Value.ToString("yyyy-MM-ddTHH:mm:ss") : null;  // Bug!!! what the hell!
```

**Comment by coolboy** [comment]
@kendomen here is the correct format
`2014-05-17T13:35:01-07:00`

**Comment by lauratomas** [comment]
@coolboy Does this mean any DateTime values must be formatted in Pacific time?

**Comment by coolboy** [comment]
nop, you can change -07:00 to any timezone you want

**Comment by lauratomas** [comment]
@coolboy I'm working with a dev who's passing this in:
`_Client.SearchManager.SearchAsync(keyword:="*.txt", updatedAtRangeFromDate:=dtFromDate, updatedAtRangeToDate:=dtToDate)`

But is getting an "invalid value" error, which looks to be an encoding issue. His timezone is +5:30. Is this expected for time zones ahead of UTC?

**Comment by lauratomas** [comment]
Bump @coolboy

**Comment by Dave-Lowndes** [comment]
Hi, I also encountered this problem - see #361 and the proposed fix #364

**Comment by SamirJikadia** [comment]
Thank you @coolboy for commenting on this #310 issue
We are using SDK for our project, Now we also found same issue
and we would like to release project early, but waiting for new release.

Is it possible to release on nuGet as early as possible.
